packages/odyssey-react-mui/src/labs/Drawer.tsx

```diff
@@ -0,0 +1,258 @@
+    odysseyDesignTokens.HueNeutralWhite};
+  padding: ${({ odysseyDesignTokens }) => odysseyDesignTokens.Spacing4};
+  align-content: center;
+  border-top: ${({ showDividers, odysseyDesignTokens }) =>
+    showDividers ? `1px solid ${odysseyDesignTokens.HueNeutral200}` : "none"};
+`;
+
+const Drawer = ({
+  ariaLabel,
+  children,
+  isOpen,
+  onClose,
+  primaryCallToActionComponent,
+  secondaryCallToActionComponent,
+  showDividers = false,
+  tertiaryCallToActionComponent,
+  testId,
+  title,
+  translate,
+  variant = "temporary",
+}: DrawerProps) => {
+  const [isContentScrollable, setIsContentScrollable] = useState(false);
+  const drawerContentRef = useRef<HTMLDivElement>(null);
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+
+  //If RTL is set in the theme, align the drawer on the left side of the screen, uses right by default.
+  const { i18n } = useTranslation();
+  const anchorDirection = i18n.dir() === "rtl" ? "left" : "right";
+
+  useEffect(() => {
+    let frameId: number;
+
+    const handleContentScroll = () => {
+      const drawerContentElement = drawerContentRef.current;
+      if (drawerContentElement) {
+        cancelAnimationFrame(frameId);
+        setIsContentScrollable(
+          drawerContentElement.scrollHeight > drawerContentElement.clientHeight,
+        );
+      }
+      frameId = requestAnimationFrame(handleContentScroll);
+    };
+
+    if (isOpen) {
+      frameId = requestAnimationFrame(handleContentScroll);
+    }
+
+    return () => {
+      cancelAnimationFrame(frameId);
+    };
+  }, [isOpen]);
+
+  const dividersVisible = useMemo(() => {
+    return showDividers || isContentScrollable;
+  }, [showDividers, isContentScrollable]);
+
+  const hasFooter = useMemo(
+    () =>
+      primaryCallToActionComponent ||
+      secondaryCallToActionComponent ||
+      tertiaryCallToActionComponent,
+    [
+      primaryCallToActionComponent,
+      secondaryCallToActionComponent,
+      tertiaryCallToActionComponent,
+    ],
+  );
+
+  return (
+    <MuiDrawer
+      data-se={testId}
+      anchor={anchorDirection}
+      open={isOpen}
+      onClose={onClose}
+      variant={variant}
+      sx={{
+        //Overrides defualt MUI inline style
+        ...(variant === "persistent" && {
+          "& .MuiDrawer-paper": {
+            transition: "none",
+          },
+        }),
+      }}
+    >
+      <DrawerContentWrapper
+        {...(isContentScrollable && {
+          //Sets tabIndex on content element if scrollable so content is easier to navigate with the keyboard
+          tabIndex: 0,
+        })}
+        odysseyDesignTokens={odysseyDesignTokens}
+        ref={drawerContentRef}
+      >
+        <DrawerHeader
+          translate={translate}
+          odysseyDesignTokens={odysseyDesignTokens}
+          showDividers={dividersVisible}
+        >
+          <Heading5>{title}</Heading5>
+          <Button
+            ariaLabel={ariaLabel}
+            label=""
```

Comment:
Any accessibility concern here with the empty label?

Reply:
good catch, this was unintentional, I don't believe we need the label here.